### DISCUSSION TITLE
成本思维

### DISCUSSION THREAD

**Discussion by giscus** [jsongo/hugo-page#1] [Announcements]
# 成本思维

成本思维 大多数看起来有所利益的事情，肯定有成本，而且有风险。风险本质上也是一种成本。\n用成本 + 风险的思维来看问题，不要只想着好处。成本和风险往往是在巨大的利益面前，特别容易被忽视的东西。\n塔勒布在《非对称风险》中表达了一个观点，他反复提到在真实的世界里，一定要风险共担，如果有人不愿意承担风险，却要享受好处，这样的人是有问题的。\n企业最大的成本，是老大的决策成本 如果动不动就来个调整，发现不对，马上再折腾几回，这损耗的是非常大的。决策成功率越低，企业就会被越来越快消磨掉。我们说大公司病，其实有一层逻辑是：公司小的时候，10 个决策中，有一两个正确的决策可能就能把营收和规模做上来；但当公司越来越大的时候，10 个决策中如果有一两个决策做错了，那都是在损耗。所以大公司病往往不是因为决策人变笨了，而是企业到了那个阶段，决策的成本变高了。\n看过一篇文章《只有10%的产品值得用AI再做一遍》，标题就直接表明了作者的观点。文中提到了 AI 时代，各大厂争先恐后地攻城略地，但后来发现，做了很多产品功能都是徒劳，浪费了很多资源。比如：\n支小宝虽有创意但操作路径复杂且多数功能未发挥 AI 能力； 如果我用支小宝坐公交，我需要把 APP 打开，点开对话，再输入指令“我要坐公交”，等待 2-3s 加载出来，这个过程至少需要 5s 以上。\n实际上，用手机的 NFC 直接刷码上车，不需要打开任何 App 就完成了这个操作。\n淘宝问问、京东京言等电商 AI 助手功能空泛、多轮对话理解能力差且 AI 划重点功能无实际价值； 饿了么的 AI 评价总结功能上线又下线，因为既无价值还浪费费用； 携程问道生成的内容干巴巴且缺乏实用性。\n这些都表明很多产品为跟上 AI 潮流而做 AI，未解决用户痛点，在未成熟阶段盲目投入，造成大师的资源浪费。\n当然也不能说探索就是浪费，只是想表达决策是需要成本的，而且经常是最大的企业成本。 制度本身也要成本 坏的制度增加企业成本，而好的制度是降低企业成本的。\n制度虽然有时让人诟病，但它们能带来一些确定性。一个集体如果制度有问题，一方面在遇到事时，经常会有模棱两可的疑惑，直到有个人做了表率，其它人慢慢地也跟上了。到后来，大家苦于不合理制度的约束，他们可能会不断地试探，直到看到不好的信号或良心觉得不妥，从而确定了一个心里底线，形成了自己的“制度”。\n刘润讲到一个案例：一家文化教育的企业，老板从来不管教学质量，这家公司里的老师能偷懒就偷懒，因为没有制度去管理他们，慢慢的好老师就都走了，留下的越来越烂，这是会传染的。\n问题制度带来的成本，就是人才流失，导致最后没有优秀的人来高效的做事，成本急剧增加。\n题外话，刘润在书中提到，对公司的管理的三种手段：\n鼓励白，即激励一些对的事； 压缩灰，即用文化作为共同的价值观，给大家一个底层的判断标准，从而遇到模棱两可的事时能有一个基础的良知判断； 禁止黑，即什么事一定不能做，这是一个更强硬的制度，防止公司烂掉。 团队的规模也有成本 我在之前的其它文章里提到了亚马逊的 Two Pizza team 原则：\n把整个系统拆分成微服务或独立模块，不同的 team 各负责一个单位的事情。Two Pizza team 是指用两块 pizza 就能吃饱的团队规模，一个业务的团队超过了这个规模，信息传递的成本就很高，沟通协作就变得困难。\n小团队沟通高效，目标也容易对齐。团队大了经常要各种对齐，很多时间花在开会上。很多会都是用来同步信息的，没有它们反而可以跑的更快。所以团队规模越来，沟通协作的成本越高，这些都是决策时要考虑的成本问题。\n机会成本 那些看似免费或便宜的东西、或能直接带来利益的事情，往往也是有机会成本的。你可能要花挺多时间在上面，研究它到底对你有没有用，是忽悠人还是可能有少许好处可以捞。\n总之不管什么样的情况，你总需要付出时间，要拿机会成本来换，而且很多并不划算，你完全可以拿这些时间来做更有价值的事情。\n忘了哪本书里提到：\n你得到的越多，选择放弃的机会成本就越大，就越不敢往前走，就越需要勇气和智慧。\n另外还有社交，也是要成本。放弃无效社交，它们会占用你原本可用于提升自己的时间。\n总之，如果尝试用成本思维去思考下，你很可能会得出不一定的决策。\n

https://www.jsongo.top/articles/thoughts/cost-thinking/

**Comment by jsongo** [comment]
这篇是 Week 43 of 2024 的 thinking，作为第一篇试水，放出来与大家交流
